src/DurableTask.AzureStorage/AzureStorageOrchestrationService.cs

```diff
@@ -1843,7 +1843,7 @@ async Task<PurgeResult> IOrchestrationServicePurgeClient.PurgeInstanceStateAsync
         /// </summary>
         /// <param name="instanceId">The orchestration instance to wait for.</param>
         /// <param name="executionId">The execution ID (generation) of the specified instance.</param>
-        /// <param name="timeout">Max timeout to wait. Only positive values are allowed</param>
+        /// <param name="timeout">Max timeout to wait. Only positive `TimeSpan` values, or `Timeout.InfiniteTimeSpan`, are allowed.</param>
```

Comment:
XML doc comments for C# have a specific convention for referencing other types.
```suggestion
        /// <param name="timeout">>Max timeout to wait. Only positive <see cref="TimeSpan"/> values or <see cref="Timeout.InfiniteTimeSpan"/> are allowed.</param>
```
If you simply want the equivalent of markdown's inline code block (`code`), then you can use `<c>code</c>`.